octavia-cli/README.md

```diff
@@ -38,10 +38,11 @@ Summary of achievements:
 
 | Date       | Milestone                           |
 |------------|-------------------------------------|
+| 2022-01-06 | Generate an API Python client from our Open API spec |
 | 2021-12-22 | Bootstrapping the project's code base |
 
 # Developing locally
-1. Install Python 3.10.0. We suggest doing it through `pyenv`
+1. Install Python 3.8.12. We suggest doing it through `pyenv`
```

Comment:
isn't the rest of the project using 3.7?

Reply:
Python 3.7 is not supported on M1. It's why I first decided to use the latest python version (3.10), but `mypy` installation is not working with 3.10, so I fell back on  3.8.
Is it a concern?

Reply:
okay. i think it's not ideal that we have different version but seems unavoidable in this case. @sherifnada  for visibility.